sdk/src/main/java/com/relaypro/sdk/Workflow.java

```diff
@@ -32,36 +32,92 @@
  */
 public abstract class Workflow implements Cloneable {
 
+    /**
+     * A decorator for a handler for the  StartEvent (workflow is starting).
```

Comment:
I don't think of these methods as a "decorator" in Java, but a callback when event occurs. You can refer users to the event javadoc for more explanation regarding the event.